### PR TITLE
Fixed Issue #63 - Added Code of Conduct

### PR DESCRIPTION
**Pull request by mittalshravika**
Added the Code of Conduct badge and spaces for headings

### CODE REVIEW

**Comment by tessel-highfive** [comment]
Thanks for the pull request, and welcome! The Rust team is excited to review your changes, and you should hear from @hipsterbrown (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. Due to the way GitHub handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

Please see [the contribution instructions](https://github.com/tessel/project/blob/master/CONTRIBUTING.md) for more information.

**Comment by Frijol** [comment]
fixes https://github.com/tessel/openwrt-tessel/issues/63

File: README.md [hunk 1]
```diff
@@ -7,7 +11,7 @@ a lightly-modified OpenWrt tree to build a Tessel-customized OpenWrt, replacing
 Building the toolchain and all system packages requires a fast Linux system and several GB of disk
 space.
 
-```
+```shell-script
```
Comment (Frijol) [inline README.md]: nice!

**Comment by Frijol** [comment]
Thanks! If you are able to do a PR like this in one commit, one Github trick is to make the commit message "fixes https://github.com/tessel/openwrt-tessel/issues/63". This will automatically link the issue to the PR, and close the issue automatically when the PR is merged.

**Pull request merged**